patches/0033-mali-Avoid-splitting-lines-across-multiple-printf-calls.patch

```diff
@@ -0,0 +1,58 @@
+mali: Avoid splitting lines across multiple printf calls
+
+Since Linux 4.9, KERN_CONT is mandatory for continued lines. See
+4bcc595ccd80 ("printk: reinstate KERN_CONT for printing continuation lines").
+In the absence of KERN_CONT, a newline may be implicitly inserted by the
+core printk code. However, KERN_CONT should only be used by core/arch
+code during early bootup as a continued line is not SMP-safe.
+
+Avoid all these issues altogether by not splitting lines across multiple
+calls.
+
+Signed-off-by: Jonathan Liu <net147@gmail.com>
+--- a/src/devicedrv/mali/common/mali_kernel_common.h
```

Comment:
@net147 how have you generated this patch? It's not git format-patch style. Can you re-generate it and update? It has to be generated --relative

Thank you

Reply:
Regenerated